x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api.ts

```diff
@@ -27,6 +27,16 @@ export async function loadAlert({
   return await http.get(`${BASE_ALERT_API_PATH}/${alertId}`);
 }
 
+export async function loadAlertNavigation({
```

Comment:
I can't tell if this function is used anymore, doesn't look like it is, wondering if it's legacy from when you tried server-side?  I didn't notice an implementation for the route either.

Reply:
Good catch! Legacy from when it resided on the server.